style.module.scss

```diff
@@ -90,6 +90,7 @@
     justify-content: space-between;
     display: flex;
     align-items: center;
+    text-align: center;
```

Comment:
This was necessary since the install overview page in the public docs unfortunately now spills into two lines for Azure. It doesn't look great, but I think this is okay-ish, and the in-app version does not suffer from this problem since its wider (see screenshot in PR description).

<img width="813" alt="Screenshot 2023-09-29 at 10 54 02 AM" src="https://github.com/pganalyze/pganalyze-docs/assets/7227/d2382af9-bf9b-4b4b-a350-7e82d699ef81">